root/static/gulpfile.js

```diff
@@ -151,9 +151,7 @@ function langToPosix(lang) {
 function buildScripts() {
   process.env.NODE_ENV = String(process.env.DEVELOPMENT_SERVER) === '1' ? 'development' : 'production';
 
-  var commonBundle = runYarb('common.js', function (b) {
-    b.expose(path.resolve(STATIC_DIR, 'lib/leaflet/leaflet-src.js'), 'leaflet');
```

Comment:
The files that need leaflet now require this path directly. `require('leaflet')` didn't work under Node.js because it couldn't resolve that name to anything (it's an alias we only add during the bundling process, at this line).